Yousei/Internal/Connectors/Internal/OnStartTrigger.cs

```diff
@@ -8,7 +8,7 @@ namespace YouseiReloaded.Internal.Connectors.Internal
     internal class OnStartTrigger : ObservableTrigger
     {
         public OnStartTrigger(IObservable<Unit> startObservable)
-            : base(startObservable.Select(_ => default(object)))
+            : base(startObservable.Select(_ => (object)_))
```

Comment:
What difference does the cast make?

Reply:
`default(object)` makes it return `null` but casting the unit value boxes it and returns it as `object`, satisfying the required type in the process.